test/controllers/v1/relay_group_membership_controller_test.exs

```diff
@@ -48,6 +46,20 @@ defmodule Cog.V1.RelayGroupMembershipControllerTest do
     assert member["name"] == relay.name
   end
 
+  test "relay group includes assigned bundles", %{authed: requestor} do
+    bundle = bundle("foo")
+    relay_group = relay_group("test-relay-group-1")
+    assign_bundle_to_group(relay_group.id, bundle.id)
+    conn = api_request(requestor, :get, "/v1/relay_groups/#{relay_group.id}")
+    fetched = json_response(conn, 200)["relay_group"]
+    assert relay_group.id == fetched["id"]
+    assert relay_group.name == fetched["name"]
+    assert length(fetched["bundles"]) == 1
+    member = Enum.at(fetched["bundles"], 0)
+    assert member["id"] == bundle.id
+    assert member["name"] == bundle.name
```

Comment:
I think all these asserts could be collapsed into one, using the match operator:

```elixir
assert %{"id" => ^relay_group_id,
         "name" => ^relay_group_name,
         "bundles" => [%{"id" => ^bundle_id,
                         "name" => ^bundle_name}]} = fetched
```
There is a downside in that things like `bundle.id` can't be in matches; you'd have to capture those in variables beforehand, but I think this gives a better sense of the shape of the data, and probably gives more usable error messages.